src/apps/testapps/testH3UniEdge.c

```diff
@@ -208,34 +209,113 @@ TEST(getH3UnidirectionalEdgesFromHexagon) {
 }
 
 TEST(getH3UnidirectionalEdgeBoundary) {
-    H3Index sf = H3_EXPORT(geoToH3)(&sfGeo, 9);
+    H3Index sf;
+    GeoBoundary boundary;
+    GeoBoundary edgeBoundary;
     H3Index* edges = calloc(6, sizeof(H3Index));
```

Comment:
Somewhat outside of your changeset, but why is `calloc` here instead of `STACK_ARRAY_CALLOC` (https://github.com/uber/h3/blob/master/src/h3lib/include/stackAlloc.h#L29)?

Reply:
Mostly likely because this was written before we introduced that. Will update.